chrome/content/helpers.js

```diff
@@ -0,0 +1,76 @@
+var isNode = (typeof process === "object");
```

Comment:
I merged some of this file in dce33b0cb5db96ccce68ce08be800681f9a0f928 so please rebase on that.